study-builder/studies/rarex/consent-self.conf

```diff
@@ -638,7 +638,7 @@
         },
         {
           "question": {
-            include required("../snippets/text-question-signature-required.conf"),
+            include required("../snippets/text-question.conf"),
```

Comment:
The "printed name" questions are not signatures, so let's use regular text question instead.